src/Converter/LineItemsConverter.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\RefundPlugin\Converter;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Sylius\Component\Core\Model\AdjustmentInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+use Sylius\Component\Core\Model\OrderItemUnitInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\RefundPlugin\Entity\LineItem;
+use Sylius\RefundPlugin\Entity\LineItemInterface;
+use Sylius\RefundPlugin\Model\UnitRefundInterface;
+use Webmozart\Assert\Assert;
+
+final class LineItemsConverter implements LineItemsConverterInterface
+{
+    /** @var RepositoryInterface */
+    private $orderItemUnitRepository;
+
+    public function __construct(RepositoryInterface $orderItemUnitRepository)
+    {
+        $this->orderItemUnitRepository = $orderItemUnitRepository;
+    }
+
+    public function convert(array $units): Collection
+    {
+        $lineItems = new ArrayCollection();
+
+        /** @var UnitRefundInterface $unitRefund */
+        foreach ($units as $unitRefund) {
+            /** @var OrderItemUnitInterface $orderItemUnit */
```

Comment:
`find` can return null as well, we can be sure it's `OrderItemUnitInterface` only after asserting it's not null.

```suggestion
            /** @var OrderItemUnitInterface|null $orderItemUnit */
```